src/main/java/amazon/HomePage.java

```diff
@@ -34,9 +34,10 @@ public HomePage(WebDriver driver) {
     protected WebElement itemTopic;
     @FindBy(xpath = "//div[@id='nav-xshop']/a[contains(text(),'Today')]")
     protected WebElement dealsOfTodayButton;
-    @FindBy(xpath = "//div[@class='a-cardui-body']")
+    @FindBy(xpath = "//div[contains(@class,'gw-col csm-placement')]")
```

Comment:
Why have you changed the selector and put a non-reliable one? What's the motivation?